src/objective/multiclass_obj.cu

```diff
@@ -65,8 +65,8 @@ class SoftmaxMultiClassObj : public ObjFunction {
     const auto ndata = static_cast<int64_t>(preds.Size() / nclass);
 
     // clear out device memory;
-    out_gpair->Reshard(GPUSet::Empty());
-    preds.Reshard(GPUSet::Empty());
+    // out_gpair->Reshard(GPUSet::Empty());
+    // preds.Reshard(GPUSet::Empty());
```

Comment:
Why are we commenting out lines? If they are not needed, we should just remove them

Reply:
Done.